trade-finance-manager-ui/server/controllers/deals/index.js

```diff
@@ -48,8 +49,7 @@ const queryDeals = async (req, res) => {
   let activeSortByField = '';
   let searchString = '';
   const { userToken } = req.session;
-
-  delete req.body._csrf;
+  const sortButtonWasClicked = (req.body.formId && req.body.formId === 'deals-table-sorting') ? true : false;
```

Comment:
Can be simplified to following statement

```javascript
 const sortButtonWasClicked = req.body?.formId === 'deals-table-sorting'
```

Reply:
Thanks, simplified.